servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/IterableMergeCompletable.java

```diff
@@ -37,7 +37,7 @@ public CompletableDynamicCountSubscriber apply(final Subscriber subscriber) {
 
     @Override
     void doMerge(final CompletableDynamicCountSubscriber subscriber) {
-        long count = 1;
```

Comment:
Why it can not be `long` anymore?

Reply:
The limit of the stack is `int` so going beyond this range will no longer be allowed.